### PR TITLE
HDDS-8527. Update Ofs (HCFS) doc to describe shared tmp dir

### PR DESCRIPTION
**Pull request by neils-dev**
## What changes were proposed in this pull request?

Update Ofs (Hadoop Compatible) document to describe the shareable `/tmp` directory for `HCFS` applications and its configuration. 

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-8527

## How was this patch tested?

CI

### CODE REVIEW

File: hadoop-hdds/docs/content/interface/Ofs.md [hunk 1]
```diff
@@ -160,12 +160,18 @@ $ ozone fs -mkdir -p /volume_1
...
 ```
 
-## Mounts
+## Mounts and Configuring /tmp
 
 In order to be compatible with legacy Hadoop applications that use /tmp/,
 we have a special temp mount located at the root of the FS.
 This feature may be expanded in the feature to support custom mount paths.
 
+Currently ozone supports two configurations for /tmp.  The first (default),
```
Comment (smengcl) [inline hadoop-hdds/docs/content/interface/Ofs.md:169]:
```suggestion
Currently Ozone supports two configurations for /tmp.  The first (default), 
```

File: hadoop-hdds/docs/content/interface/Ofs.md [hunk 2]
```diff
@@ -175,8 +181,9 @@ $ ozone sh volume create tmp
 $ ozone sh volume setacl tmp -al world::a
 ```
 
-These commands only needs to be done **once per cluster**.
+These commands only need to be done **once per cluster**.
 
+### For /tmp directory per user (default)
```
Comment (smengcl) [inline hadoop-hdds/docs/content/interface/Ofs.md:186]:
```suggestion
### For /tmp directory per user (default)

```

File: hadoop-hdds/docs/content/interface/Ofs.md [hunk 3]
```diff
@@ -192,6 +199,31 @@ directory. e.g.:
 $ ozone fs -touch /tmp/key1
 ```
 
+### For a sharable /tmp directory common to all users
```
Comment (smengcl) [inline hadoop-hdds/docs/content/interface/Ofs.md:202]:
```suggestion
### For a sharable /tmp directory common to all users

```

**Review by smengcl — APPROVED** [review]
Thanks @neils-dev for the doc addition

**Comment by smengcl** [comment]
Thanks @neils-dev for the doc addition!

**Comment by neils-dev** [comment]
Thanks @smengcl .

**Pull request merged**